src/main/java/com/vmware/vipclient/i18n/util/LocaleUtility.java

```diff
@@ -101,19 +101,13 @@ public static boolean isDefaultLocale(Locale locale) {
     }
 
     /*
-     * Judge if a locale string is English locale string.
+     * Check if a language tag matches the default locale
```

Comment:
Check if a language tag matches the default locale -> Check if a language tag matches the default locale from configuration file